xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -194,17 +196,28 @@ int CPVRRecordings::GetNumRecordings()
   return m_recordings.size();
 }
 
-int CPVRRecordings::GetRecordings(CFileItemList* results)
+bool CPVRRecordings::HasDeletedRecordings()
+{
+  CSingleLock lock(m_critSection);
+  return m_bHasDeleted;
+}
+
+int CPVRRecordings::GetRecordings(CFileItemList* results, bool deleted)
```

Comment:
I think we better put these deleted recordings in it's own vector instead of in the same one as the standard recordings, so we don't need these checks. especially after https://github.com/xbmc/xbmc/pull/6207 has been merged in, after which we can simplify this further